packages/forklift-console-plugin/src/modules/Plans/views/details/components/PlanWarningCondition.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Linkify from 'react-linkify';
+
+import { Alert, Text, TextContent, TextVariants } from '@patternfly/react-core';
+
+export const PlanWarningCondition: React.FC<{
```

Comment:
no need for direct export and default export, you can remove the export from here